contrib/storage-druid/README.md

```diff
@@ -4,13 +4,13 @@ Drill druid storage plugin allows you to perform SQL queries against Druid datas
 This storage plugin is part of [Apache Drill](https://github.com/apache/drill)
 
 ### Tested with Druid version
-[0.16.0-incubating](https://github.com/apache/incubator-druid/releases/tag/druid-0.16.0-incubating)
+[0.22.0](https://github.com/apache/druid/releases/tag/druid-0.22.0)
```

Comment:
Are there any versions of Druid that this will NOT work with?   If so, maybe say that this will work with Druid > n or whatever.... I'm not very knowledgable of Druid, so I don't really know if that is even relevant.

Reply:
Druid 0.17.x to 0.19.x does not have the "offset" parametser in the scan query api, so paging is not possible. I thought it would be better to avoid all Druid Versions < 0.20.x for scan api instead of sacrifice pagination and reading all rows at once.

This means:

- Drill < 1.20 works well with Druid <= 0.16 (select api)
- Drill >= 1.20 will work with Druid >= 0.20 (scan api)
- Druid 0.17 to 0.19 will not be supported by Drill because of missing offset parameter in scan api

I mentioned this in the docs https://github.com/apache/drill-site/pull/18/files#diff-c47da4191a5cd392dabb624042159b63be990b216242e3da3491896378025e24R45

Reply:
Perfect.  Thx!